src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -12688,15 +12708,18 @@ .maxstack 8
         }
 
         [Theory, WorkItem(44902, "https://github.com/dotnet/roslyn/issues/44902")]
-        [InlineData(false)]
-        [InlineData(true)]
-        public void CopyCtor(bool useCompilationReference)
+        [InlineData(false, TargetFramework.Standard)]
+        [InlineData(false, TargetFramework.NetCoreApp)]
```

Comment:
>TargetFramework.NetCoreApp [](http://example.com/codeflow?start=27&length=26)

Doing this is probably going to create a problem when a test is running on desktop. We will take advantage of covariant returns at build, but the runtime won't support that when we try to execute the test environment. I suggest always using TargetFramework.StandardLatest target and simply adjust test expectations accordingly based on the platform that executes the unit-test. There is a ```RuntimeUtilities.IsCoreClrRuntime``` API to detect that, for example.

Reply:
@AlekseyTs From other tests that used .NET Standard only, they never had runtime support for new features. See #53416 where the tests I'm modifying there had an unreachable code path. (See #53413 where the test succeeded when I was throwing an exception in the unreachable code path).

Reply:
>  From other tests that used .NET Standard only, they never had runtime support for new features. 

There is a difference between TargetFramework.Standard and TargetFramework.StandardLatest, I believe.

Reply:
Thanks @AlekseyTs!